webapp/src/app.tsx

```diff
@@ -4741,6 +4741,9 @@ document.addEventListener("DOMContentLoaded", () => {
     else if (pxt.BrowserUtils.isIpcRenderer()) workspace.setupWorkspace("idb");
     else if (pxt.BrowserUtils.isPxtElectron()) workspace.setupWorkspace("fs");
     else workspace.setupWorkspace("browser");
+    // disable auth in iframe scenarios
+    if (workspace.getWorkspaceType() === "iframe")
```

Comment:
In the future we might want to allow auth but disable cloud sync for the iframe workspace.

Reply:
yeah.